bpaingest/libs/sensitive_data.py

```diff
@@ -0,0 +1,176 @@
+import re
+import glob
+import os.path
+import shapely
+import shapely.geometry
+import fiona
+from ..util import csv_to_named_tuple
+from ..util import make_logger
+from .ingest_utils import get_clean_number
+
+logger = make_logger(__name__)
+
+SENSITIVE_DATA_DIR = "sensitive_data"
+
```

Comment:
I don't think this code will pass `flake8`, could you lint before you merge? :)

Reply:
hm it does actually:lee@xublr:~/src/bpa-ingest$ flake8 bpaingest/libs/sensitive_data.py 
lee@xublr:~/src/bpa-ingest$